interp/stdarg.mli

```diff
@@ -37,6 +37,8 @@ val wit_int_or_var : (int or_var, int or_var, int) genarg_type
 
 val wit_ident : Id.t uniform_genarg_type
 
+val wit_identref : lident uniform_genarg_type
```

Comment:
This is only used in VERNAC EXTEND afaict, so there is no need for the toplevel value type. It could be set to `Empty.t` or `Id.t`. The last thing I want is to wantonly start using locations in the runtime representation of data that shouldn't be located, so better prevent this statically.

Reply:
This is used in `TACTIC EXTEND` too (especially after #13103, e.g. in `instantiate`, since the initial objective, #11842, was to merge the entries `ident` and `identref`, so as to facilitate the pedagogy).

I'm ok to have it to `Id.t` though.